AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -279,6 +280,8 @@ + (void)scheduleNodeForRecursiveDisplay:(ASDisplayNode *)node
       for (ASDisplayNode *node in displayingNodes) {
         [node __recursivelyTriggerDisplayAndBlock:NO];
       }
+      [[NSNotificationCenter defaultCenter] postNotificationName:ASRenderingEngineDidDisplayScheduledNodesNotification
```

Comment:
Do we need away to check if a specific node is in the set that displayed? Maybe we can actually pass the displaying nodes object as the object, and any listeners can check for contains object?